ipn/ipnserver/server.go

```diff
@@ -1184,3 +1193,47 @@ func findTrueNASTaildropDir(name string) (dir string, err error) {
 	}
 	return "", fmt.Errorf("shared folder %q not found", name)
 }
+
+func loadExtraEnv() (env []string, err error) {
+	if runtime.GOOS != "windows" {
+		return nil, nil
+	}
+	name := filepath.Join(os.Getenv("ProgramData"), "Tailscale", "tailscaled-env.txt")
```

Comment:
Rather than an absolute path can we find this relative to the executable, for debug setups that run from a directory rather than using the installer?

Reply:
Not worth it. If you're using a debug setup and running a custom binary from cmd.exe, you can set the environment variable by hand already in cmd.exe or powershell. You don't need this mechanism for that.

This is purely for running as a service.